kibbeh/public/locales/hi/translation.json

```diff
@@ -119,17 +119,17 @@
     "bottomVoiceControl": {
       "leaveCurrentRoomBtn": "वर्तमान रूम छोड़े",
       "confirmLeaveRoom": "क्या आप छोड़ना चाहते हैं?",
-      "leave": "Leave",
+      "leave": "विदा",
```

Comment:
this doesn't make sense, instead of leave this translates to "i am leaving" or "bye, im going"